pkg/kubectl/cmd/create_clusterrole.go

```diff
@@ -133,6 +134,12 @@ func (c *CreateClusterRoleOptions) Validate() error {
 				return fmt.Errorf("invalid verb: '%s' for nonResourceURL", v)
 			}
 		}
+
+		for _, nonResourceURL := range c.NonResourceURLs {
+			if !strings.HasPrefix(nonResourceURL, "/") {
```

Comment:
add validate of '*', does it neccesary to validate '/foo/*bar' ?
the server side doesn't validate the nonResourceURL, should we validate in the server side too? @liggitt